examples/DropletCollision/dropColl_MD_setup.py

```diff
@@ -0,0 +1,446 @@
+    
+    print(f'   Real radius: {radius_real}')
+    
+    # Calculate distance of particle from center
+    rx_dist = chp['rx'] - 0.5*xBoxLength
+    ry_dist = chp['ry'] - 0.5*yBoxLength
+    rz_dist = chp['rz'] - 0.5*zBoxLength
+    chp['distance'] = (rx_dist*rx_dist + ry_dist*ry_dist + rz_dist*rz_dist)**0.5
+    
+    # Particles within droplet (radius plus approx. interface thickness (more or less arbitrarily set to 3.0))
+    mask = chp['distance'] < (radius_real + 3.0)
+    
+    # Assign velocity
+    chpLeft['vy']  = chpLeft['vy'].mask(mask, chpLeft['vy'] + 0.5*v_rel)
+    chpRight['vy'] = chpRight['vy'].mask(mask, chpRight['vy'] - 0.5*v_rel)
+    
+    # Change componentid to make postprocessing easier
+    chpLeft['cid']  = chpLeft['cid'].mask(mask, 2)
+    chpRight['cid'] = chpRight['cid'].mask(mask, 3)
+    
+    # Cut both droplet checkpoints and keep a distance of 1 sigma between both
+    sizeCutted = 0.5*yBoxLength + radius + 0.5*distance - 0.5  # Size in y direction of single box when cutted
+    chpLeft = chpLeft[chpLeft['ry'] <= sizeCutted]
+    chpRight = chpRight[chpRight['ry'] >= (yBoxLength - sizeCutted)]
+    
+    # Shift right checkpoint
+    chpRight.loc[:,'ry'] = chpRight['ry'] + 2*radius + distance
+    
+    chpCollision= pd.concat([chpLeft,chpRight], axis=0, ignore_index=True)
+    
+    # Refresh particle ids
+    chpCollision['pid'] = np.arange(len(chpCollision)) + 1
+    
+    
+    headerXML.find('headerinfo/length/x').text = str(boxSize[0])
+    headerXML.find('headerinfo/length/y').text = str(boxSize[1])
+    headerXML.find('headerinfo/length/z').text = str(boxSize[2])
+    headerXML.find('headerinfo/number').text = str(len(chpCollision))
+    headerXML.find('headerinfo/time').text = str(0.0)
+    
+    headerXMLTree.write(out_file_path_header)
+    exp.exp_chp_bin_DF(out_file_path, chpCollision)
+    
+    return  1
+
+
+
+
+#%% Main program which can be called from the command line
+if __name__ == '__main__':
+    
+    
+    #%% Initial conditions
+    
+    radius = 30        # Approximate radius of droplet
+    v_rel = 2.0        # Relative velocity of droplets; each droplet gets 0.5*v_rel assigned
+    temperature = 0.7  # Initial temperature of system
+    distance = 10      # Distance between droplets measured between surfaces
+    
+    
+    # Other
+    num_reps = 10
+    timestep = 0.003647347
+    
+    radius_shrink = 1.00  # Cut out with larger radius to compensate shrinking due to surface tension
+    
+    # For reproducibility
+    random.seed(10)
+    
+    #%% Computer settings
+    
+    # Path to folder where simulation files will be created
+    work_folder = 'PATH'
+    work_folder = os.path.join(work_folder, f'dropColl_r{radius}_v{str(v_rel).replace(".", "-")}_T{str(temperature).replace(".", "-")}')
+    
+    # Path to ls1 executable
+    ls1_exec = 'PATH/build/src/MarDyn'
+    
+    # Path to templates
+    template_folder = 'PATH/templates'
```

Comment:
So if I understand correctly, this is the "configuration" part of the script, right? Seems a bit random to throw this somewhere in the middle of a 450 line script. I'd suggest making these input arguments to the script, with input validation. This way, the user directly gets feedback if they forgot to set anything.

An alternative would be to visually significantly highlight the setup section with something like:
```python
###########################################################################
############################## CONFIGURATION ##############################
###########################################################################
```